examples/simple-game-server/Makefile

```diff
@@ -32,6 +32,8 @@ WINDOWS_VERSIONS = ltsc2019
 WINDOWS_DOCKER_PUSH_ARGS = # When pushing set to --push.
 # Build with Windows support
 WITH_WINDOWS=1
+# Build with ARM64 support
+WITH_ARM64=1
```

Comment:
Please increment the `server_tag` for the image from `0.12` to `0.13` since we are making changes to the image (e.g. adding the arm build).